SharpBucket/V2/EndPoints/IssuesResource.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SharpBucket.Utility;
+using SharpBucket.V2.Pocos;
+
+namespace SharpBucket.V2.EndPoints
+{
+    public class IssuesResource
+    {
+        private readonly string _slug;
+        private readonly string _accountName;
+        private readonly RepositoriesEndPoint _repositoriesEndPoint;
+
+        public IssuesResource(string accountName, string repoSlugOrName, RepositoriesEndPoint repositoriesEndPoint)
+        {
+            _accountName = accountName.GuidOrValue();
+            _slug = repoSlugOrName.ToSlug();
+            _repositoriesEndPoint = repositoriesEndPoint;
+        }
+
+        /// <summary>
+        /// List open issues on the repository.
+        /// </summary>
+        /// <returns></returns>
+        public List<Issue> ListIssues()
+            => ListIssues(new ListParameters());
+
+        /// <summary>
+        /// List issues on the repository.
+        /// </summary>
+        /// <param name="parameters">Parameters for the query.</param>
+        public List<Issue> ListIssues(ListParameters parameters)
+        {
+            if (parameters == null)
+                throw new ArgumentNullException(nameof(parameters));
+            return _repositoriesEndPoint.ListIssues(_accountName, _slug, parameters);
+        }
+
+        /// <summary>
+        /// Enumerate open issues on the repository.
+        /// </summary>
+        public IEnumerable<Issue> EnumerateIssues()
+            => EnumerateIssues(new EnumerateParameters());
+
+        /// <summary>
+        /// Enumerate issues on the repository.
+        /// </summary>
+        /// <param name="parameters">Parameters for the queries.</param>
+        public IEnumerable<Issue> EnumerateIssues(EnumerateParameters parameters)
+        {
+            if (parameters == null)
+                throw new ArgumentNullException(nameof(parameters));
+            return _repositoriesEndPoint.EnumerateIssues(_accountName, _slug, parameters);
+        }
+
+        #region Issue Resource
+
+        /// <summary>
+        /// Get the Issue Resource.
+        /// BitBucket does not have this Resource so this is a "Virtual" resource
+        /// which offers easier manipulation of a specific Issue.
```

Comment:
This two comment lines "BitBucket does not have ... specific Issue" seems copy/paste from elsewere and is not valid in this case.
The Issue Resource is well documented [here](https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/issues/%7Bissue_id%7D), it's not a virtual resource at all.

note: it seems to be copy paste from pullRequestsResource where I don't understand either why we speak about virtual resource there. maybe a copy paste from API V1 ?